docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,133 @@
+For example, to install Kyma from a specific version, such as `1.19.1`, run:
+
+    ```bash
+    kyma alpha deploy --source=1.19.1
+    ```
+
+- Alternatively, to build Kyma from your local sources and deploy it on a remote cluster, run:
+
+    ```bash
+    kyma alpha deploy --source=local
+    ```
+    > **NOTE:** By default, Kyma expects to find the sources in the `./workspace folder`. To adjust the path, set the `-w ${PATH_TO_KYMA_SOURCES}` parameter.
+
+- To deploy Kyma with only specific components, run:
+
+    ```bash
+    kyma alpha deploy --components {COMPONENTS_FILE_PATH}
+    ```
+    `{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed. In the following example, only these 6 components are deployed on the cluster:
+
+    ```bash
+    defaultNamespace: kyma-system
+    prerequisites:
+      - name: "cluster-essentials"
+      - name: "istio"
+        namespace: "istio-system"
+    components:
+      - name: "testing"
+      - name: "xip-patch"
+      - name: "istio-kyma-patch"
+      - name: "dex"
+    ```
+
+- To override the standard Kyma installation, run:
+
+    ```bash
+    kyma alpha deploy --values-file {OVERRIDE_FILE_PATH}
+    ```
+    `{OVERRIDE_FILE_PATH}` is the path to a YAML file containing the parameters to override.<br>
+In the following example, overrides are provided for 2 different components: `ory` and `monitoring`.
+ 
+    - For `ory`, the values of `hydra.deployment.resources.limits.cpu` and `hydra.deployment.resources.requests.cpu` will be overriden to `153m` and `53m` respectively.
+    - For `monitoring`, the values of `alertmanager.alertmanagerSpec.resources.limits.memory` and `alertmanager.alertmanagerSpec.resources.requests.memory` will be overriden to `304Mi` and `204Mi` respectively.
+    ```bash
+     ory:
+      hydra:
+        deployment:
+          resources:
+            limits:
+              cpu: 153m
+            requests:
+              cpu: 53m
+     monitoring:
+      alertmanager:
+        alertmanagerSpec:
+          resources:
+            limits:
+              memory: 304Mi
+            requests:
+              memory: 204Mi
+    ```
+
+- You can also provide multiple override files at the same time:
+
+    ```bash
+    kyma deploy --values-file {OVERRIDEL_FILE_1_PATH} --values-file {OVERRIDE_FILE_2_PATH}
+    ```
+
+- Alternatively, you can specify single values instead of a file:
+
+    ```bash
+    kyma deploy --value ory.hydra.deployment.resources.limits.cpu=153m \
+    --value ory.hydra.deployment.resources.requests.cpu=53m \
+    --value monitoring.alertmanager.alertmanagerSpec.resources.limits.memory=304Mi \
+    --value monitoring.alertmanager.alertmanagerSpec.resources.requests.memory=204Mi
+    ```
+
+## Upgrade Kyma
+
+To upgrade the Kyma version on the cluster, you also use the `alpha deploy` command as described in section [Install Kyma](#install-kyma).
```

Comment:
I hope I didn't misunderstand the meaning, but "you also use the alpha deploy command..." sounds a bit confusing (is this an alternative way, the only way, or a required additional step in the upgrade process?) Maybe we can either say "you can also use" if it's an alternative way, or simply say "use", if it's the only way to do it? 
```suggestion
To upgrade the Kyma version on the cluster, you can also use the `alpha deploy` command as described in the [Install Kyma](#install-kyma) section.
```

Reply:
Good point. The meaning was that the deploy command isn't only used for installation (as in the section above) but also for upgrading. So, `deploy` is the only way to upgrade, but upgrading is not the only function of `deploy`. I'll rephrase it.

Reply:
The entire section is going to be extended with more examples.